src/inputs/Table/components/TableToolbar/TableToolbar.js

```diff
@@ -4,15 +4,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { useStyles } from './style';
-import { FullscreenButton } from '../FullscreenButton';
+import { FullscreenButton, ImportButton, ExportButton } from '..';
 
 export const TableToolbar = (props) => {
-  const { isFullscreen, toggleFullscreen } = props;
+  const { isFullscreen, toggleFullscreen, isReady, onImport, onExport, editMode, customToolbarActions, labels } = props;
   const classes = useStyles();
 
   return (
-    <div className={classes.tableToolbar}>
-      <FullscreenButton isFullscreen={isFullscreen} toggleFullscreen={toggleFullscreen} />
+    <div className={classes.tableToolbar} style={isReady ? { borderBottom: 'none' } : null}>
+      <FullscreenButton
+        isFullscreen={isFullscreen}
+        toggleFullscreen={toggleFullscreen}
+        disabled={!isReady}
+        label={labels.fullscreen}
```

Comment:
This label is required in `FullscreenButton` but not in `TableToolbar`: you must provide a default "fallback" value
Example: `label={ labels.fullscreen ?? 'Fullscreen' }`

Reply:
DONE